packages/mux-viewer-count/README.md

```diff
@@ -0,0 +1,63 @@
+<p align="center">
+  <h1 align="center">&lt;mux-viewer-count/&gt;</h1>
+  <a href="https://npmcharts.com/compare/@mux/mux-viewer-count?interval=30"><img src="https://img.shields.io/npm/dm/@mux/mux-viewer-count.svg?sanitize=true" alt="Downloads"></a>
+    <a href="https://www.npmjs.com/package/@mux/mux-viewer-count"><img src="https://img.shields.io/npm/v/@mux/mux-viewer-count.svg?sanitize=true" alt="Version"></a>
+    <a href="https://www.npmjs.com/package/@mux/mux-viewer-count"><img src="https://img.shields.io/npm/l/@mux/mux-viewer-count.svg?sanitize=true" alt="License"></a>
+</p>
+
+# Introduction
+
+`<mux-viewer-count></mux-viewer-count>` is a Mux-flavored HTML5 viewer count element.
+
+This element shows the current number of viewers for the specified video.
+
+# Installation
+
+If you're using `npm` or `yarn`, install that way:
+
+## Package manager
+
+```
+yarn add @mux/mux-viewer-count
+```
+
+or
+
+```
+npm i @mux/mux-viewer-count
+```
+
+Then, import the library into your application with either `import` or `require`:
+
+```js
+import '@mux/mux-viewer-count';
+```
+
+or
+
+```js
+require('@mux/mux-viewer-count');
+```
+
+## CDN option
+
+Alternatively, use the CDN hosted version of this package:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@mux/mux-viewer-count@0"></script>
+```
+
+If you are using ecmascript modules, you can also load the `mux-viewer-count.mjs` file with `type=module`:
+
+```html
+<script type="module" src="https://cdn.jsdelivr.net/npm/@mux/mux-viewer-count@0/dist/mux-viewer-count.mjs"></script>
+```
+
+## Usage
+
+`<mux-viewer-count>` has two attributes. The first is called `token`, and it should be a signed JavaScript Web Token (JWT)
+for which you want the viewer count. (For more information on creating a JWT for a particular video, see 
+https://docs.mux.com/guides/data/see-how-many-people-are-watching.
+
+The second attribute is called `pollInterval`. It specifies the number of seconds that the component should wait between
```

Comment:
We had _many_ _many_ long convos about attr naming on Media Chrome deciding between

- concatenated, no space attrs like `pollinterval` (which is what [HTML spec](https://www.w3.org/TR/design-principles/#casing-rules) says it should be). As I call it `smushedcase`
- the more readable `poll-interval` (kebab case)

MC decided to follow HTML

The jury is still out if Mux Elements will do that too, or not. But as of today except for a few exceptions Mux Elements does `kebab-case` (if we choose for Elements to go smushedcase it will be a large breaking change that we probably wouldn't do for a while).

For the sake of consistency between elements, I think this should be `poll-interval`. Either way, we're living with `kebab-case` now on everything else, so I think it's worth staying consistent.

If we make the decision for all elements to move to smushedcase in the future, we should move them all over at the same time to minimize the inconsistency.

Reply:
I'll make that change!